docs/vm-ee-guide.md

```diff
@@ -0,0 +1,324 @@
+              nephe.io/namespace=sample-ns
+              nephe.io/owner-vm=vm1-11610
+              nephe.io/owner-vm-vpc=test-vnet-13196
+              nephe.io/tag-Cloud=azure
+              nephe.io/tag-Name=vm1
+Annotations:  <none>
+API Version:  crd.antrea.io/v1alpha2
+Kind:         ExternalEntity
+Metadata:
+  Creation Timestamp:  2023-04-17T19:50:18Z
+  Generation:          1
+  Managed Fields:
+    API Version:  crd.antrea.io/v1alpha2
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:labels:
+          .:
+          f:nephe.io/cloud-region:
+          f:nephe.io/cloud-vm-name:
+          f:nephe.io/cloud-vm-uid:
+          f:nephe.io/cloud-vpc-name:
+          f:nephe.io/cloud-vpc-uid:
+          f:nephe.io/kind:
+          f:nephe.io/namespace:
+          f:nephe.io/owner-vm:
+          f:nephe.io/owner-vm-vpc:
+      f:spec:
+        .:
+        f:endpoints:
+        f:externalNode:
+    Manager:         nephe-controller
+    Operation:       Update
+    Time:            2023-04-17T19:50:18Z
+  Resource Version:  556954
+  UID:               f0309a24-632b-4ebd-bbeb-9b205151a311
+Spec:
+  Endpoints:
+    Ip:           10.0.0.4
+    Ip:           20.245.116.185
+  External Node:  nephe-controller
+Events:           <none>
+```
+
+### Labels
+
+| Label                   | Description                                                                               | Azure                                                            | AWS                                              |
+|-------------------------|-------------------------------------------------------------------------------------------|------------------------------------------------------------------|--------------------------------------------------|
+| nephe.io/cloud-vm-uid   | Unique identifier of VM on cloud                                                          | VM `vmId` property on cloud                                      | Instance ID of VM on cloud                       |
+| nephe.io/cloud-vpc-uid  | Unique identifier of VPC on cloud                                                         | VNET `resourceGuid` property on cloud                            | VPC ID of VPC on cloud                           |
+| nephe.io/cloud-vpc-name | User assigned VPC name on cloud                                                           | VNET `name` property on cloud                                    | VPC `Name` tag on cloud                          |
+| nephe.io/cloud-vm-name  | User assigned VM name on cloud                                                            | VM `name` property on cloud                                      | VM `Name` tag on cloud                           |
+| nephe.io/owner-vm       | VM object Name within Kubernetes cluster                                                  | VM `name` property suffixed with ascii sum of VM Resource ID     | Instance ID of VM on cloud                       |
```

Comment:
Name = name

Reply:
In AWS, Name tag is displayed with uppercase "N"